common/djangoapps/third_party_auth/models.py

```diff
@@ -473,8 +467,6 @@ class SAMLConfiguration(ConfigurationModel):
     )
     site = models.ForeignKey(
         Site,
-        null=True,
```

Comment:
This is no longer here. None of the site keys are nullable, but they default to the site specified in settings.SITE_ID.